integration_test/gce/gce_testing.go

```diff
@@ -1302,8 +1302,12 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 	return vm, nil
 }
 
-func IsSUSE(imageSpec string) bool {
-	return strings.HasPrefix(imageSpec, "suse-") || strings.HasPrefix(imageSpec, "opensuse-") || strings.Contains(imageSpec, "sles-")
+func IsSLES(vm_OS_ID string) bool {
+	return vm_OS_ID == "sles" || vm_OS_ID == "sles_sap"
+}
+
+func IsSUSE(vm_OS_ID string) bool {
+	return vm_OS_ID == "opensuse" || vm_OS_ID == "opensuse-leap" || vm_OS_ID == "sles" || vm_OS_ID == "sles_sap"
```

Comment:
this can call IsSLES, to reduce redundant code, e.g.
return vm_OS_ID == "opensuse" || vm_OS_ID == "opensuse-leap" || IsSLES(VM_OS_ID)

Reply:
+1

Reply:
Done.